.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: false
 language: go
 
 go:
-  - 1.9
+  - 1.11
```

Comment:
The default Go version in the buildpack is Go 1.10. I think we should pin the Go version in both places to 1.11.